app/src/main/java/de/danoeh/antennapod/fragment/FeedItemlistFragment.java

```diff
@@ -252,7 +252,9 @@ public void onCreateOptionsMenu(@NonNull Menu menu, @NonNull MenuInflater inflat
         optionsMenu = menu;
         FeedMenuHandler.onCreateOptionsMenu(inflater, menu);
         iconTintManager.updateTint();
-        MenuItemUtils.setupSearchItem(menu, (MainActivity) getActivity(), feedID);
+        if (feed != null) {
+            MenuItemUtils.setupSearchItem(menu, (MainActivity) getActivity(), feedID, feed.getTitle());
+        }
```

Comment:
If the feed is null, the menu items should still be setup. Just the feed title can be left out. That prevents possible flickering when menu items are displayed/hidden for some feeds.